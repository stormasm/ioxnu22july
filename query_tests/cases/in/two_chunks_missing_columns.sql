-- Basic query tests
-- IOX_SETUP: TwoChunksMissingColumns

-- query data
SELECT * from "table" order by time;
