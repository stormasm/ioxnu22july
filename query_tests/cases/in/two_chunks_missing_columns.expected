-- Test Setup: TwoChunksMissingColumns
-- SQL: SELECT * from "table" order by time;
+--------+--------+--------+------+------+------+--------------------------------+
| field1 | field2 | field3 | tag1 | tag2 | tag3 | time                           |
+--------+--------+--------+------+------+------+--------------------------------+
| 10     | 11     |        | a    | b    |      | 1970-01-01T00:00:00.000000100Z |
| 20     |        | 22     | a    |      | c    | 1970-01-01T00:00:00.000000200Z |
+--------+--------+--------+------+------+------+--------------------------------+
